Servers/UI/OJS.Servers.Ui/ClientApp/src/components/modal/ContestModal.tsx

```diff
@@ -1,42 +1,42 @@
-import React, { useCallback } from 'react';
+import React, { useCallback, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import Box from '@mui/material/Box';
 import Modal from '@mui/material/Modal';
 import Typography from '@mui/material/Typography';
 
-import { IContestModal } from '../../common/types';
+import { IContestModalInfoType } from '../../common/types';
 import { useAppUrls } from '../../hooks/use-app-urls';
 import { useCurrentContest } from '../../hooks/use-current-contest';
 import Button, { ButtonSize, ButtonType } from '../guidelines/buttons/Button';
 
 import styles from './ContestModal.module.scss';
```

Comment:
Overall the text in the modal is very large

Reply:
Fixed.